apps/api/views.py

```diff
@@ -31,3 +32,4 @@ def get_elevation(lat, lng):
 
 
 calculatecoordinate = CalculateCoordinate.as_view()
+
```

Comment:
you can remove this blank line :lipstick:

Reply:
this is a pep8 recomendation

Reply:
some files in this projects don't follow `W292`, but no problem, this is aesthetic